apps/code-examples/src/app/code-examples/forms/checkbox/inline-help/demo.component.html

```diff
@@ -8,6 +8,7 @@
             <sky-help-inline
               class="sky-control-help"
               (actionClick)="onActionClick()"
+              ariaLabel="More information about Email input"
```

Comment:
```suggestion
              ariaLabel="Information about Email"
```
We don't need "input" here, just like we don't have identifiers above in the corresponding ARIA labels for data grid, data entry grid and tree view. I think the screen reader supplies something, so this would be redundant. (I considered suggesting that we add the missing article ("the") before "Email," but I think we can leave it out of an ARIA label just like we would in a label in the UI.) ... I'll make suggestions with this change for the remaining items as well.